web/docs/operators/metrics.md

```diff
@@ -298,6 +298,23 @@ per schema.
 | `schema_id`   | `string` | The schema ID of the batch.                                   |
 | `events`      | `uint64` | The amount of events that were retrieved from the `topic`.    |
 
+### `tenzir.metrics.tcp-listen`
```

Comment:
For the user, there is no distinction between `from tcp` and `load tcp`, even though there is on under the hood. The `tcp-listen` operator is entirely hidden from the user.

I think we should make two further changes here:
1. Emit the same metric in the `tcp` loader
2. Find a name for the metric that's easier to understand for the user

Reply:
This was updated to just `tcp` and is emitted for all TCP connections now.